book/doctrine.rst

```diff
@@ -144,6 +140,60 @@ for you:
         # Version 5.5.3 introduced "utf8mb4", which is recommended
         collation-server     = utf8mb4_general_ci # Replaces utf8_general_ci
         character-set-server = utf8mb4            # Replaces utf8
+        
+    You can also change the defaults for Doctrine so that the generated SQL
+    uses the correct character set.
+    
+    .. configuration-block::
+        .. code-block:: yaml
+    
+            doctrine:
+                dbal:
+                    default_table_options:
+                        charset: utf8mb4
+                        collate: utf8mb4_unicode_ci
+
+        .. code-block:: xml
+
+            <!-- app/config/config.xml -->
+            <?xml version="1.0" encoding="UTF-8" ?>
+            <container xmlns="http://symfony.com/schema/dic/services"
+                xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                xmlns:doctrine="http://symfony.com/schema/dic/doctrine"
+                xsi:schemaLocation="http://symfony.com/schema/dic/services
+                    http://symfony.com/schema/dic/services/services-1.0.xsd
+                    http://symfony.com/schema/dic/doctrine
+                    http://symfony.com/schema/dic/doctrine/doctrine-1.0.xsd">
+
+                <doctrine:config>
+                    <doctrine:dbal
+                        driver="%database_driver%"
+                        host="%database_host%"
+                        dbname="%database_name%"
+                        user="%database_user%"
+                        password="%database_password%">
+                            <doctrine:default-table-option name="charset">utf8</doctrine:default-table-option>
+                            <doctrine:default-table-option name="collate">utf8_unicode_ci</doctrine:default-table-option>
```

Comment:
and we can omit the other options here to be consistent with the YAML example (same below in the PHP config)